src/service/api/category.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const {Router} = require(`express`);
+const {HttpCode} = require(`../../constants`);
+
+const route = new Router();
+
+module.exports = (app, service) => {
```

Comment:
```suggestion
module.exports = (app, service) => {
const route = new Router();
```
сделай объявление роутера прям в функции (во всех)

Reply:
сделано